src/renderers/dom/shared/dangerousStyleValue.js

```diff
@@ -23,9 +25,10 @@ var isUnitlessNumber = CSSProperty.isUnitlessNumber;
  *
  * @param {string} name CSS property name such as `topMargin`.
  * @param {*} value CSS property value such as `10px`.
+ * @param {ReactDOMComponent} component
  * @return {string} Normalized style value with dimensions applied.
  */
-function dangerousStyleValue(name, value) {
+function dangerousStyleValue(name, value, component) {
```

Comment:
Eeeh, now I'm having mixed feelings about this one.  If we're going to make a change, we might as well add the name of the current component in addition to the owner component (to provide more context).  So directly passing in something like `component._currentElement._owner.getName()` and `component.._tag`.  We could do that, but it also means wiring it through `createMarkupForStyles` and I'm not sure it's worth it at this point.

In a perfect world, we would have the full path, which would require merging https://github.com/facebook/react/pull/5167 first, but we need to decide on the fate of that PR before we can merge it, and it might not be worth delaying this PR even longer.

I'm tempted to say that this is fine for now, and we can make it better after https://github.com/facebook/react/pull/5167.  Unless @spicyj wants it changed now (or unless we merge https://github.com/facebook/react/pull/5167 real fast), my intuition is that maybe you just ignore this comment and we deal with stateless components and/or giving the full parent path when those things actually happen.